example 1:

public class Main {
    public static void main(String[] args) {
        try {
            int[] myNumbers = {1, 2, 3};
            System.out.println(myNumbers[10]);
        } catch (Exception l) {
            int[] myNumbers = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
            System.out.println(myNumbers[2]); 
        }
    }
}

example 2:

public class Main {
    public static void main(String[] args) {
        try {
            int[] myNumbers = {1, 2, 3};
            System.out.println(myNumbers[10]);
        } catch (Exception e) {
            System.out.println("something went wrong."); 
        }finally{
            System.out.println("the 'try catch' is finished.");
        }
    }
}


example 3: (using string)

public class Main {
    public static void main(String[] args) {
        try {
            String[] Brothers = {"Devesh","Aswin","Lokesh"};
            System.out.println(Brothers[2]);
        } catch (Exception e) {
           // int[] myNumbers = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
            System.out.println("something went wrong."); 
        }finally{
            System.out.println("the 'try catch' is finished.");
        }
    }
}
